recipes-bsp/k3-rti-wdt/k3-rti-wdt_1.1.bb

```diff
@@ -26,5 +26,5 @@ do_prepare_build() {
 }
 
 dpkg_runbuild_prepend() {
-    export CC="gcc"
+    sbuild_export CC="gcc"
```

Comment:
As already commented on upstream: this naming is wrong. Don't spread it to downstream layers, fix it in Isar first.